packages/battery/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright 2017, the Flutter project authors. All rights reserved.
```

Comment:
Please make sure this exactly matches https://github.com/flutter/flutter/blob/master/LICENSE